error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -122,6 +123,21 @@ Stream<T> after(T[] array) {
     }
   }
 
+  /**
+   * Prefer {@link Arrays#asList(Object[])} over {@link Stream#toList()} as the former is clearer.
+   */
+  static final class ArraysAsList<T> {
+    @BeforeTemplate
+    List<T> before(@NotMatches(IsRefasterAsVarargs.class) T[] array) {
+      return Arrays.stream(array).toList();
```

Comment:
Let's add an `XXX` to track that we're losing unmodifiability.

(We could also target `ImmutableList#copyOf`. For now I'll just make a note about this.)